tests/test_message.py

```diff
@@ -1,5 +1,8 @@
 import pytest
 import linkatos.message as message
+import os
+
+BOT_ID = os.environ.get("BOT_ID")
```

Comment:
As I said before, this is bad. Besides that, once your function receives the id, you can suddently test it without relying in the context which gives you isolation and predictability.